src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/RazorCompletionEndpoint.cs

```diff
@@ -137,14 +138,14 @@ public bool CanResolve(CompletionItem completionItem)
                 {
                     case RazorCompletionItemKind.DirectiveAttribute:
                     case RazorCompletionItemKind.DirectiveAttributeParameter:
+                    case RazorCompletionItemKind.TagHelperAttribute:
                         return true;
                 }
 
                 return false;
             }
 
-            if (completionItem.IsTagHelperElementCompletion() ||
-                completionItem.IsTagHelperAttributeCompletion())
```

Comment:
Why are attribute completions no longer valid here?

Reply:
It's encompassed on line 141 now where we added the case for `RazorCompletionItemKind.TagHelperAttribute`